### DISCUSSION TITLE
Zoom Events Callback

### DISCUSSION THREAD

**Discussion by infinite-looper** [Jericho/ZoomNet#116] [Ideas]
Hi,

It would be nice if ZoomNet provided the resources for Zoom callback events (meeting started, etc.). Is this something you plan to add at some point?

**Comment by Jericho** [comment]
I already started thinking about adding this feature and I even have a ticket for it #78 

To be honest, I kinda put it on the back burner because I didn't know if there would be any interest for it. I guess the fact that you are asking about it tells me that at least one person is interested!

**Reply by infinite-looper** [reply]
Yes! I've got a bunch of webhooks to write...the ZoomNet library or an add-on package could go as far as provide a reference endpoint (ASP.NET controller) to fully abstract the processing of callback events.

**Reply by Jericho** [reply]
@infinite-looper  Wanted to give you a quick update. I'm making slow progress, but progress nonetheless. I published two alpha releases so far and the latest includes the ability to parse the "app_deautorize" event, which I believe is particularly important for you.

**Reply by infinite-looper** [reply]
Yes thanks a bunch!
We will definitely take it for a ride. Priorities for the Zoom integration have changed recently, but we plan to complete it early next week.

**Reply by Jericho** [reply]
I'm going to go ahead and release this new functionality. ZoomNet will be able to parse the App Deauthorize event, the "meeting" events and the "webinar" events.
